.github/workflows/ci.yml

```diff
@@ -359,8 +359,14 @@ jobs:
           submodules: "recursive"
       - uses: ./.github/actions/cleanup
       - uses: ./.github/actions/setup-rust
-      # TODO(ngates): we should run tests here when they pass!
-      - run: ./gradlew shadowJar
+      # Build vortex-jni shared library, and copy into JAR.
```

Comment:
perhaps this should be its own xtask command.

CodaHale (the guy who made dropwizard and dw metrics) has a nice setup where the entire CI process runs via the xtask binary: https://github.com/codahale/yellhole/blob/main/xtask/src/xtask.rs

This is also cool b/c it lets you test run the CI locally in a single command. Out of scope I think for this PR but might be nice to switch to